community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PeriodicCommitAcceptanceTest.scala

```diff
@@ -122,7 +122,7 @@ class PeriodicCommitAcceptanceTest extends ExecutionEngineFunSuite
     val (_, txCounts) = executeAndTrackTxCounts(queryText)
 
     // then
-    txCounts should equal(TxCounts(commits = 3, rollbacks = 0))
```

Comment:
This test used to commit after 9 rows, then after 19 rows, and then finally when the execution was over.
Now it commits after 10 rows, and then when the execution is over (after 20 rows).